terraform/modules/gsp-cluster/main.tf

```diff
@@ -22,8 +22,10 @@ variable "codecommit_url" {
   type = "string"
 }
 
+data "aws_caller_identity" "current" {}
```

Comment:
I'm not sure this is the best way / the right way / a good idea at all. It might be better to have this module receive the account id as a variable.